remove-hooks.ps1

```diff
@@ -0,0 +1,53 @@
+Function Clean-Location {
+    Param(
+        [Parameter(Mandatory=$True)]
+        [string]$location
+    )
+    Begin {
```

Comment:
If you are using a begin block where are the process and end block?

Reply:
removed the begin block